Source/Diagnostics/ComputeDiagFunctors/ComputeDiagFunctor.H

```diff
@@ -29,8 +29,8 @@ public:
 
     virtual void PrepareFunctorData ( int /* i_buffer */, bool /* ZSliceInDomain */,
                                       amrex::Real /* current_z_boost */,
-                                      amrex::Box /* buffer_box */, const int /* k_index_zlab */) {};
-    virtual void InitData() {};
+                                      amrex::Box /* buffer_box */, const int /* k_index_zlab */) {}
```

Comment:
It's potentially sub-ideal to remove the param names since this member function should better be having a doxygen string, too.
We could think about making the function purely virtual instead. All deriving classes are then forced to truly implement it, which can also be the empty implementation.

Or, if we want to spare the force-implementation downstream, we can use the `amrex::ignore_unused` here in the function body.